org-code-javabuilder/lib/src/main/java/org/code/javabuilder/BaseTestRunner.java

```diff
@@ -0,0 +1,77 @@
+package org.code.javabuilder;
```

Comment:
note: this file is basically a copy/paste from the previous state of `TestRunner`, with an additional parameter for `StatusMessageKey`